### PR TITLE
DOC: updated load docstrings

### PR DESCRIPTION
**Pull request by aburrell**
Updated the load docstrings so that variable names in the docs reflect those in the code.

Fixed in response to https://github.com/pysat/pysat/pull/587, which did not require any local updates.

## Type of change

Please delete options that are not relevant.

- This change requires a documentation update

# How Has This Been Tested?

No code changed.

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

**Comment by rstoneback** [comment]
Tests failing on 
`CondaValueError: invalid package specification: numpy==`

**Comment by aburrell** [comment]
Yup.  Unit tests are also failing, though, so I'm working on that.

File: pysatModels/tests/test_utils_match.py [hunk 1]
```diff
@@ -229,8 +230,11 @@ def test_tinc_success(self, tinc_val, num):
         assert len(self.out.data.data_vars[self.ref_col]) == num
         assert np.all(np.isfinite(
             self.out.data.data_vars[self.ref_col].values))
+        assert self.out.index[0].date() == self.stime.date()
+        print(self.out.index[-1])
```
Comment (rstoneback) [inline pysatModels/tests/test_utils_match.py:234]:
```suggestion

```

**Review by rstoneback — APPROVED** [review]
Thanks @aburrell! Minor cleanup suggestion included.

File: pysatModels/tests/test_utils_match.py (hunk 1 above)
Comment (aburrell) [reply pysatModels/tests/test_utils_match.py:234]: Good catch!

**Pull request merged**